Parboil/resources/lbm/main.h

```diff
@@ -0,0 +1,59 @@
+/* $Id: main.h,v 1.1 2008/03/04 17:30:03 stratton Exp $ */
+
+/*############################################################################*/
+
+#pragma once
+
+/*############################################################################*/
+
+#include <stdio.h>
+#include <time.h>
+#include <chrono>
+#include "config.h"
+#include "parboil.h"
+
+/*############################################################################*/
+
+typedef struct
+{
+	float timeScale;
+	clock_t tickStart, tickStop;
+	struct tms timeStart, timeStop;
```

Comment:
`struct tms` is most likely a bug, include `time.h` [declares a struct tm but not tms](https://pubs.opengroup.org/onlinepubs/007908799/xsh/time.h.html).